build.yaml

```diff
@@ -55,11 +55,18 @@ build-env:
     cd /
 
     # build lxc
+    apk add meson ninja docbook2x docbook2x-doc curl
     git clone --depth 1 ${{LXC_CLONE_URL}} -b ${{LXC_BRANCH}}
     cd lxc
-    ./autogen.sh
-    ./configure --enable-static-binaries --prefix=/usr
-    make -j$(grep -c processor /proc/cpuinfo) install
+    mkdir /etc/default
+    # patches
+    ## static lib name
+    curl -L https://github.com/lxc/lxc/commit/64eb31d02d4933ad414239130df3dff45a0f6f91.diff | git apply
```

Comment:
This looks odd.  
Something like it would be necessary if we were building a static commit, but since we're trackign a branch, I put up https://github.com/lxc/lxc/pull/4225 to get that into the stable-5.0 branch.

So, if that lands, it will actually break builds here.